stable/artifactory/templates/artifactory-installer-info.yaml

```diff
@@ -10,16 +10,13 @@ metadata:
 data:
   installer-info.json: |
     {
-      "productId": "JFrogInstaller_Helm_{{ template "artifactory.chart" . }}",
+      "productId": "Helm_artifactory/{{ .Chart.Version }}",
       "features": [
       {
-        "featureId": "AppVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
+        "featureId": "ArtifactoryVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
       },
       {
-        "featureId": "Installer/{{ default "helm" .Values.installer.type }}"
-      },
-      {
-        "featureId": "Platform/{{ default "kubernetes" .Values.installer.platform }}"
```

Comment:
@eldada Keep this as its used in different fork of chart repo for other platforms like GKE-MP, Rancher, IBM